spec.mediawiki

```diff
@@ -535,6 +511,10 @@ The successful response will contain a unique order ID that must be referenced f
 |-
 |}
 
+Market order cannot be cancelled by the client.
```

Comment:
Market orders cannot be cancelled. (depending on how we want to phrase this, we could add "since a market order never becomes a standing order" or "since cancel orders only affect orders on the book")